hack/build-go.sh

```diff
@@ -30,5 +30,5 @@ fi
 
 mkdir -p ${BIN_PATH}
 
-echo "Building ${REPO}/cmd/${WHAT} (${VERSION})"
-CGO_ENABLED=0 GOOS=${GOOS} GOARCH=${GOARCH} go build ${GOFLAGS} -ldflags "${GLDFLAGS}" -o ${BIN_PATH}/${WHAT} ${REPO}/cmd/${WHAT}
+echo "Building ${REPO}/cmd/${WHAT} (${VERSION_OVERRIDE})"
```

Comment:
I made a quick PR to fix it here: https://github.com/openshift/machine-config-operator/pull/192

Reply:
Looks like we did the same thing....